pkg/core/validators/messages.go

```diff
@@ -0,0 +1,19 @@
+package validators
+
+import (
+    "fmt"
+    "strings"
+)
+
+func MustHaveOnlyOne(entity string, allowedValues ...string) string {
+    return fmt.Sprintf(`%s must have only one type defined: %s`, entity, strings.Join(allowedValues, ", "))
+}
+
+func MustBeDefined() string {
```

Comment:
nit: why it's a func instead of const?

Reply:
I first wrote `MustHaveOnlyOne` and then that one but it turned out it did not need any arguments so I left it as it is. Can change to const but I guess it does not hurt.

Reply:
done